src/error.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use thiserror::Error;
+
+/// Triggered when is an issue parsing an SVG file.
```

Comment:
I'm not sure this doc string is well formed

Reply:
Ah, good spot. Will fix.